doc/dune-files.rst

```diff
@@ -586,10 +586,9 @@ to use the :ref:`include_subdirs` stanza.
 
 - ``(modes <modes>)`` is for modes which should be built by default. The
   most common use for this feature is to disable native compilation
-  when writing libraries for the OCaml toplevel. It can also be used to define
-  libraries that should be compiled with Melange (see :ref:`melange_main`). The
-  following modes are available: ``byte``, ``native``, ``best``, and ``melange``.
-  ``best`` is ``native`` or ``byte`` when native compilation isn't available.
+  when writing libraries for the OCaml toplevel. The following modes are
+  available: ``byte``, ``native`` and ``best``. ``best`` is ``native`` or
```

Comment:
For some reason, it's not allowing me to use the suggest mode. 

This just needs a comma after `native` in the first sentence as part of the series: `byte`, `native`, and `best`.

Reply:
(Oh it's already merged! That's why I couldn't use the suggest mode!)